index.html

```diff
@@ -39,77 +39,312 @@
+          <section class="grid">
+            <section class="grid1"></section>
+            <section class="grid2">
+              <br />
+              <h2>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h2>
+              <br />
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+              <br />
+              <button>See Project</button>
+              <br />
+            </section>
+          </section>
+            <section class="grid">
+            <section class="grid1"></section>
+            <section class="grid2">
+              <br />
+              <h2>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h2>
+              <br />
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+              <br />
+              <button>See Project</button>
+              <br />
+            </section>
+          </section>
+          <section class="grid">
+            <section class="grid1"></section>
+            <section class="grid2">
+              <br />
+              <h2>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h2>
+              <br />
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+              <br />
+              <button>See Project</button>
+              <br />
+            </section>
+          </section>
+          <section class="grid">
+            <section class="grid1"></section>
+            <section class="grid2">
               <br />
-              <h1>Hi, I'm Dagmawi</h1>
-              <h2>I’m a Software Developer</h2>
+              <h2>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h2>
+              <br />
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+              <br />
+              <button>See Project</button>
+              <br />
+            </section>
+          </section>
+          <section class="grid">
+            <section class="grid1"></section>
+            <section class="grid2">
+              <br />
+              <h2>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h2>
+              <br />
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+              <br />
+              <button>See Project</button>
+              <br />
```

Comment:
- [ ] Please make sure all your project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:
![image](https://user-images.githubusercontent.com/92303891/219874714-8d504a0a-bd6e-410e-b994-a69b35368504.png)